src/index.ts

```diff
@@ -68,13 +115,15 @@ const htmlPlugin: PluginFn = (qiankunName, microOption) => {
 
   return {
     name: 'qiankun-html-transform',
-    configResolved (config) {
-      isProduction = config.command === 'build' || config.isProduction
+    enforce: 'pre',
```

Comment:
After adjusting the execution order of the plugins, it will not work in the production environment.